src/mlpack/bindings/go/print_defn_output.hpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file print_defn_output.hpp
+ * @author Yasmine Dumouchel
+ *
+ * Print the output for the definition in a Python binding .go file for a given
```

Comment:
Oops, this should be `Go` also not `Python`.

Reply:
Updated.